renku/api/datasets.py

```diff
@@ -452,3 +454,11 @@ def check_for_git_repo(url):
         except InvalidGitRepositoryError:
             is_git = False
     return is_git
+
+
+def get_dataset_metadata_paths():
```

Comment:
why does this need to be a function? it could just be a list.

Reply:
well, since same code is used on multiple places (it was more than 2 like it is now) I decided to extract it as a function  cause of DRY'ed approach

Reply:
yes I understand... but it doesn't need to be a function. 

```
DATASET_METADATA_PATHS = [
        Path(RENKU_HOME) / Path(DatasetsApiMixin.DATASETS),
        Path(RENKU_HOME) / Path(LinkReference.REFS),
]
```